packages/react/slider/src/Slider.tsx

```diff
@@ -465,19 +376,29 @@ const SliderPart = React.forwardRef((props, forwardedRef) => {
           event.preventDefault();
         }
       })}
-      /**
-       * Prevent pointer events on other elements on the page while sliding.
-       * For example, stops hover states from triggering on buttons if
-       * mouse moves over a button during slide.
-       *
-       * Also ensures that slider receives all pointer events after mouse down
-       * even when mouse moves outside the document.
-       */
       onPointerDown={composeEventHandlers(props.onPointerDown, (event) => {
-        (event.target as HTMLElement).setPointerCapture(event.pointerId);
+        const target = event.target as HTMLElement;
+        target.setPointerCapture(event.pointerId);
+        // Prevent browser focus behaviour because we focus a thumb manually when values change.
+        event.preventDefault();
+        // Touch devices have a delay before focusing so won't focus if touch immediately moves
+        // away from target (sliding). We want thumb to focus regardless.
+        if (context.thumbs.has(target)) {
```

Comment:
Checking context instead of using the `isThumb` method. This allowed me to remove the `data-radix-slider-thumb` attribute that we had left over because of that `isThumb` util.